core/src/main/java/com/airbnb/aerosolve/core/models/NDTreeModel.java

```diff
@@ -42,6 +43,32 @@ public NDTreeModel(List<NDTreeNode> nodeList) {
     this(nodeList.toArray(new NDTreeNode[nodeList.size()]));
   }
 
+  public double stdDivByMinLeafNodesCount() {
```

Comment:
it is std/min,  higher means more diff
no reference, if diff is not too much, ndtree is similar to spline.